packages/vitest/rollup.config.js

```diff
@@ -71,37 +70,56 @@ export default ({ watch }) => [
               .filter(i => !['src', 'index', 'dist', 'node_modules'].some(j => i.includes(j)) && i.match(/^[\w_-]+$/))),
           )
           if (parts.length)
-            return `chunk-${parts.slice(-2).join('-')}.[hash].mjs`
+            return `chunk-${parts.slice(-2).join('-')}.[hash].js`
         }
-        return 'vendor-[name].[hash].mjs'
+        return 'vendor-[name].[hash].js'
       },
     },
     external,
     plugins: [
       ...plugins,
+      {
```

Comment:
I think we can just put this files in root?

Reply:
That's possible too if you don't mind the two extra files in root. I can change that.

Reply:
Updated 👍 I kept the `defineConfig` since it provides types. But let me know if you want to minimize the changes here.

Reply:
I'm fine with `defineConfig`, good job 👍🏻